docs/source/dev-setup/devenv.rst

```diff
@@ -153,7 +180,7 @@ few commands.
 
 If those commands completely successfully, you're ready to Go!
 
-If you plan to use the Hyperledger Fabric application SDKs then be sure to check out their prerequisites in the Node.js SDK `README <https://github.com/hyperledger/fabric-sdk-node#build-and-test>`__ and Java SDK `README <https://github.com/hyperledger/fabric-gateway-java/blob/main/README.md>`__.
+If you plan to use the Hyperledger Fabric application SDKs then be sure to check out their prerequisites in the Node.js SDK `README <https://github.com/hyperledger/fabric-sdk-node#build-and-test>`__ and Java SDK `README <https://github.com/hyperledger/fabric-gateway-java/blob/master/README.md>`__.
```

Comment:
Keep as main

Reply:
It was backported with reset of the code. Thank you for notifying me. I will edit it .. after searching the DCO check issue.